src/panels/config/logs/dialog-download-logs.ts

```diff
@@ -39,7 +39,7 @@ class DownloadLogsDialog extends LitElement {
 
   private _dialogClosed() {
     this._dialogParams = undefined;
-    this._lineCount = 100;
+    this._lineCount = 500;
```

Comment:
```suggestion
    this._lineCount = DEFAULT_LINE_COUNT;
```
It can be nice to define the constant at the top to use it in multiple place.